src/link.c

```diff
@@ -733,24 +742,50 @@ LinkTable *LinkTable_disk_open(const char *dirn)
 
 LinkTable *path_to_Link_LinkTable_new(const char *path)
 {
-    Link *link = path_to_Link(path);
-    LinkTable *next_table = link->next_table;
+    struct Link *link = NULL, *tmp_link = NULL;
+    struct Link linkcpy = {0};
+    LinkTable *next_table = NULL;
+    if (!strcmp(path, "/")) {
+        next_table = ROOT_LINK_TBL;
+        linkcpy = *next_table->links[0];
+        tmp_link = &linkcpy;
+    } else {
+        link = path_to_Link(path);
+        tmp_link = link;
+        LinkTable *next_table = link->next_table;
```

Comment:
I am pretty sure this line does absolutely nothing, because the shadowed variable goes out of scope immediately.

Reply:
Yup the compiler says the variable is unused....